### ISSUE TITLE
[Accessibility][ConfigPanelThresholds]  Combobox controls rely on option text as a label

### ISSUE THREAD

**Issue by jmbuss** [awslabs/iot-app-kit#2513]
**Describe the bug**
**Standard:** 3.3.2 Labels or Instructions (A)
**Affected Groups:** Cognitively Impaired, Screen Reader Users

**Notes and Advice** 
The "Arithmetic Operator" custom combobox controls rely upon option text as a visible label:
 
<select>
<option disabled>selectable option</option>
<option>1</option>
...
</select>
 
Once an option is selected the control label will no longer be visible.
 Recommendation 
All controls must always have a persistent, visible label (in some cases implicit labels are acceptable).
 
For a HTML <select> element:
 
<label for="octo">selectable option</label>
<select id="octo">
...
</select>
 
For a custom control:
 
<span id="octo">selectable option</span>
<div role="combobox" aria-labelledby="octo">
...
</div>
 Resources 
[Understanding 3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
[How to Meet 3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/quickref/#labels-or-instructions)

**Acceptance Criteria**
The arithmetic operator should have a label above it using cloudscape which says Operator
